src/run.ts

```diff
@@ -214,55 +292,29 @@ export async function runVersion({
     q: searchQuery,
   });
   let changedPackages = await getChangedPackages(cwd, versionsByDirectory);
+  let changedPackagesInfo = (
+    await Promise.all(
```

Comment:
nit: previously this was parallelized, we could maintain this with:
```suggestion
    Promise.all(
```